DorisProject/src/app/pages/add-category/add-category.component.ts

```diff
@@ -1,7 +1,10 @@
-import { Component, OnInit } from '@angular/core';
-import { FormArray, FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
-import { Router } from '@angular/router';
-import { CategoriasService } from 'src/app/service/categorias.service';
+import {Component, OnInit} from '@angular/core';
+import {FormArray, FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
+import {Router} from '@angular/router';
+import {CategoriasService} from 'src/app/service/categorias.service';
+import {ModalComponent} from '../modal/modal.component';
```

Comment:
Codacy has a fix for the issue: [Replace `ModalComponent` with `·ModalComponent·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
import { ModalComponent } from '../modal/modal.component';
```